Merge feature/merchant index/story 38 into master

Capleugh opened the pull request:
**Functionality:**
-  Update merchants table to include a column titled "enabled?"
- Create button in admin's merchant index view to disable a merchant.

**Testing:**
- working in local development
- 100% coverage

**Related Issues:**
#86

Capleugh committed:
Create test for adding disable button beside merchants on admin/merchants

Capleugh committed:
Update admin merchant index view to include disable button

Capleugh committed:
Update merchants table to include enabled column

Capleugh committed:
Add spacing in strong params

Capleugh committed:
Add sad path testing

Capleugh committed:
Create patch request for admin/merchants path

Capleugh committed:
Create update action in admin merchants controller

Capleugh committed:
Create conditional for whether or not disable button displays

Capleugh committed:
Remove strong params from admins merchant controller

Capleugh committed:
More robust testing

mintona commented on app/controllers/admin/merchants_controller.rb:
```diff
@@ -6,4 +6,18 @@ def index
+
+  private
+    def change_status_of_merchant(merchant)
+      merchant.update(enabled?: false)
```
Discussed using toggle in next branch.

Capleugh committed:
Co-authored-by: David Barriga-Gomez 
<50917634+DavidBarriga-Gomez@users.noreply.github.com>
Co-authored-by: Alison Vermeil 
<51250305+mintona@users.noreply.github.com>
Co-authored-by: Harrison Levin 
<29346170+hale4029@users.noreply.github.com>

Capleugh committed:
Fix accidental deletion of test

mintona commented on spec/features/admin/dashboard_spec.rb:
```diff
@@ -70,8 +70,8 @@
-      
-      within "#all-orders" do
+
+      within "#all-orders" do 
         expect(page.body.index("#{order_4.id}")).to be < page.body.index("#{order_2.id}")
         expect(page.body.index("#{order_2.id}")).to be < page.body.index("#{order_3.id}")
         expect(page.body.index("#{order_3.id}")).to be < page.body.index("#{order_1.id}")
```
This was fixing an accidental deletion. Reviewed in person and looks good.

mintona reviewed (approved):
Reviewed with Carleigh and David in person. Merging.
